files/configs/horizon-source/templates/horizon-source-template.yml

```diff
@@ -0,0 +1,27 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@ load("@ytt:json", "json")
+#@ horizon = json.decode(data.values.config)["ESCAPED_HORIZON_SERVER"]
+#@ horizonTls = json.decode(data.values.config)["HORIZON_DISABLE_TLS"]
+
+apiVersion: sources.tanzu.vmware.com/v1alpha1
+kind: HorizonSource
+metadata:
+  name: horizon-source
+spec:
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1
+      kind: Broker
+      name: default
+      namespace: vmware-functions
+  address: #@ "https://"+horizon
+  #@ if horizonTls == "True":
```

Comment:
Just to double-check: `if horizonTls == "True"` means disable TLS or enable TLS?

Reply:
That's correct, I most likely kept the variable name "simple" when working on vSphere sources from event-router, so just dup'ed variable name here but it is indeed reading from https://github.com/vmware-samples/vcenter-event-broker-appliance/blob/development/files/setup.sh#L34 

Happy to modify for clarity purposes

Reply:
Yeah, would be great to have those two align, i.e. if `horizonTls == "True"` `skipTLSVerify` should be `false`

Reply:
Resolved